deploy/tests/test.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+# Copyright (c) 2016 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+KUBE_GLUSTER_DEAMONSET=../kube-templates/glusterfs-daemonset.yaml 
+KUBE_HEKETI_DEPLOYMENT=../kube-templates/heketi-deployment.yaml
+KUBE_HEKETI_SA=../kube-templates/heketi-service-account.yaml 
+OCP_GLUSTER_TEMPLATE=../ocp-templates/glusterfs-template.yaml
+OCP_HEKETI_TEMPLATE=../ocp-templates/heketi-template.yaml
+OCP_HEKETI_SA=../ocp-templates/heketi-service-account.yaml
+TOPOLOGY=../topology.json.sample
+
+# syntax checking can not be done if file has more than one yaml context 
+# so split and call check
+# check kube-templates/heketi-deployment.yaml to understand what I am talking about
+split_yaml_check() {
```

Comment:
It should be possible to write this function so that the case of a single context (even without opening "---\n") would be a valid corner case. So the caller should not need to know and just call this function, ideally under a different name, like yaml_syntax_check(...) or so...